Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -600,31 +600,27 @@ public extension String {
         guard index >= 0 && index < count else { return nil }
         return self[self.index(startIndex, offsetBy: index)]
     }
+    
+    /// SwifterSwift: Safely subscript string within a close or half-open range.
```

Comment:
There are other types of ranges
```suggestion
    /// SwifterSwift: Safely subscript string within a given range.
```